.eslintrc.json

```diff
@@ -2,12 +2,14 @@
     "env": {
         "browser": true,
         "commonjs": true,
-        "node": true
+        "node": true,
+        "es2021": true
```

Comment:
Essa parte aqui deu problema pra mim e não sei o que ta rolando
`Error: .eslintrc.json:  Environment key "es2021" is unkown`

Reply:
Opa, foi esse mesmo erro que deu com a Thais!
Se você conseguir investigar um pouco, seria muito bom. Tenta mudar para `es2020`, buscar o que significa esse valor no `"env"` e coisas do tipo

Reply:
Fazendo uma pesquisa rápida não achei muito sobre esse erro, mas pode ser por causa da versão do ESLint